TYPO3.Neos/Resources/Private/Styles/Foundation/_labels-badges.scss

```diff
@@ -7,7 +7,7 @@
 .neos-label,
 .neos-badge {
   display: inline-block;
-  padding: 2px 4px;
+  padding: 4px 6px;
```

Comment:
by changing this, you need to ensure all labels and badges used in the whole backend still fit their spot.. would definitely not recommend doing such a change without really testing it thoroughly.. if you need bigger labels for this particular case, then find a way to do that specifically..